src/main/java/com/example/springbootbookshop/entity/Book.java

```diff
@@ -33,4 +38,11 @@ public class Book {
     @Column(nullable = false)
     private boolean isDeleted = false;
 
+    @ManyToMany
```

Comment:
pls be careful with `@Data` annotation, since it includes role, and in some cases it may be unnecessary